bpf/lib/nat.h

```diff
@@ -327,8 +343,6 @@ snat_v4_nat_handle_mapping(struct __ctx_buff *ctx,
 		if (IS_ERR(ret))
 			return ret;
 
-		set_v4_rtuple(tuple, *state, &rtuple);
-		*state = __snat_lookup(map, &rtuple);
```

Comment:
Can you explain why this lookup is no longer needed :pray: ? It's not obvious to me.

Reply:
I was just reading the comment below that it was OK to fail a deletion? If that's the case, maybe we can always try to delete RevSNAT if the SNAT exists.  But I am not sure about the penalty of deletion failure. Adding it back.